##(*It's not all about*) location, location, location: an old saying goes out of style in cancer research
Written by Alyssa Kaiser

Imagine for a moment that you’re an oncologist, a doctor specializing in cancer diagnosis and treatment. You’re assigned ten patients: five with lung cancer and five with pancreatic cancer. Now, I want you to group these patients into two categories for treatment.
 
The answer seems obvious - how else would you separate these ten patients, if not by tumor location? The lungs of patient A are more similar to the lungs of patient B then they are to the pancreas of patient B. Logically, this trend should continue when comparing lung and pancreatic tumors, which has been the assumption underlying cancer research and therapy. However, as the technology for dissecting the makeup of individual cancers has become more available and widely used, we are beginning to understand that classifying cancers based on tissue location does not necessarily reflect the underlying mutational landscape of the individual tumors.
 
What do I mean by “*mutational landscape*”? Before we move on, we first need to understand what cancer is and how it arises. Cancer is a group of diseases caused by abnormal growth and division (proliferation) of cells. Under normal conditions, cells exist in a balance between proliferation and death so that the total number of cells remains approximately equal. In cancer, this balance is skewed and cell division takes over, resulting in abnormal growth of tissues, or tumors.
 
This equilibrium between division and death is disrupted by events called mutations, which are changes to a cell’s DNA. DNA contains the information needed to build the cell’s machinery that keeps it (and you) alive. Some of these components are small molecules called proteins, which perform most biological processes in your body. A change to a protein caused by a mutation can have devastating effects on that protein’s function in the cell and may promote uncontrolled growth. To better understand this concept, think of a cell as a car and all of the car’s parts are its proteins. If I tape down the gas pedal, which would be analogous to a mutation, the car would accelerate uncontrollably. Importantly, the same outcome of uncontrollable acceleration can be produced in more than one way; a problem with the engine or brakes could lead to a similar acceleration.
                                                   		
However, unlike the car analogy, which requires one “mutation” to achieve uncontrollable behavior, cells are much more difficult to perturb because they have built in mechanisms to try to control cell division and growth. Thus, several mutations are often required for a cell to become a cancer cell (Figure 1). These combinations of mutations comprise the “mutational landscape” I referred to earlier. This simply refers to the mutations in a cancer cell that cause it to grow and divide at abnormal rates. 

> ![](https://stanfordhealthcare.org/content/dam/SHC/clinics/stanford-cancer-genetics/images/stanfordcancergenetics-diagram-cellmutation.jpg)
Figure 1: [Multiple mutations are needed to create a cancer cell. 
source](https://stanfordhealthcare.org/medical-clinics/cancer-genetics-program/understanding-genetics/how-genes-cause-cancer.html)

As technology advances, identifying a tumor’s mutations is becoming more and more accessible. We can now take a sample from a patient’s tumor and identify which mutations in its DNA are likely drivers of abnormal growth. To do this, we sequence DNA isolated from cancer cells. DNA-sequencing is a method that “[reads](https://www.genome.gov/10001177)” all the letters in a cell’s DNA. With this information, we are able to identify similarities and differences between tumors besides the tissue in which they’re located.
 
A recent study by Ciriello et al. took an in depth look at the similarities and differences of the DNA sequences among tumors from a variety of tissues. Overall, 12 cancer types were represented. The authors analyzed 3,299 patient tumors and classified these cancers based on their mutational profiles. The trends they discovered have changed the paradigm of how doctors and scientists view cancer. The analysis grouped cancers into distinct categories based on what mutations were present in their tumors, and these groupings were made independent of the tissue of tumor origin. Cancers from the same tissue would only group together if their mutational landscapes were similar. The resulting groups were composed of tumors with similar genetic mutations but diverse locations of origin. Essentially, the authors discovered that tumor location was not necessarily an indicator of tumor similarity, and that tumors originating at different tissues could be more similar to one another than tumors originating the same tissue. 
 
What does this mean for the cancer treatment? These data reveal the importance of considering the genetic makeup of cancer in addition to its tissue location when prescribing targeted cancer therapies. While some treatments are designed for tumors in a specific location (e.g. surgery, [topical treatments](http://archderm.jamanetwork.com/article.aspx?articleid=479198)), many cancer drugs are designed to combat specific proteins that drive cell growth and proliferation. Historically, these drugs are distributed among cohorts of patients chosen by tumor location. To illustrate the problem this creates, let’s create a hypothetical scenario. Imagine you had a drug that targeted protein A, which was necessary for growth of a lung cancer. However, not all lung cancers will rely on protein A for growth, making the drug ineffective against these cancer cells. But some pancreatic cancers may also rely on protein A, making these tumors an ideal target for the drug. When you only consider tissue location for the administration of the drug, you may miss other patients that could benefit from a therapy.

The authors propose that genetic alterations in a tumor should be considered in treatment decisions. Classification of tumors based on their mutational landscapes will identify novel therapeutic options. Previously developed drugs will find new life as their use is extended beyond their initial patient cohort to treat cancers originating from a diverse range of tissues. This moves the field closer to “personalized medicine,” where tumors are assessed at the individual level to determine the best treatment (Figure 2).

![](https://pct.mdanderson.org/img/home.jpg)
Figure 2: [Personalized cancer therapy](https://pct.mdanderson.org/#/) 

This paradigm shift has already reaped benefits. Researchers have discovered that some gastric (stomach) cancers have mutations in proteins that repair the cell’s DNA. A subset of breast and colorectal cancers share the same mutations, and drugs called [PARP inhibitors](http://www.cancer.gov/publications/dictionaries/cancer-terms?cdrid=660869) have been developed that specifically target these cells. Gastric cancer, the third leading cause of cancer-related deaths, has proven particularly challenging to treat. PARP inhibitors may provide therapeutic relief to some patients suffering from gastric cancer. The study was published too recently (September 2015) for clinical trials to have been conducted, but it will be exciting to learn how effective these inhibitors are in the treatment of gastric cancer.

1. Ciriello G, Miller ML, Aksoy BA, Senbabaoglu Y, Schultz N, Sander C. Emerging landscape of oncogenic signatures across human cancers. Nature genetics. 2013;45(10):1127-1133. doi:10.1038/ng.2762.
 
2. Alexandrov LB, Nik-Zainal S, Siu HC, Leung SY, Stratton MR. A mutational signature in gastric cancer suggests therapeutic strategies. Nat Commun. 2015 Oct 29;6:8683. doi: 10.1038/ncomms9683. 
